roboteam_ai/src/ApplicationManager.cpp

```diff
@@ -119,7 +119,8 @@ void ApplicationManager::runOneLoopCycle() {
             }
         }
         strategy = BTFactory::getTree(BTFactory::getCurrentTree());
-
+        static int i=0;
```

Comment:
I disagree

Reply:
me too

Reply:
it's a testing thing i forgot to remove. Calm your tits.